gym/wrappers/test_atari_preprocessing.py

```diff
@@ -1,24 +1,25 @@
 import numpy as np
 import gym
-import pytest
 from gym.wrappers import AtariPreprocessing
-try:
-    import atari_py
-except ImportError:
-    atari_py = None
+import pytest
+pytest.importorskip('atari_py')
 
-@pytest.mark.skipif(atari_py is None, reason='Only run this test when atari_py is installed')
 def test_atari_preprocessing():
     import cv2
-    env1 = gym.make('PongNoFrameskip-v0')
-    env2 = AtariPreprocessing(env1, screen_size=84, grayscale_obs=True)
-    env3 = AtariPreprocessing(env1, screen_size=84, grayscale_obs=False)
-    obs1 = env1.reset()
-    assert obs1.shape == (210, 160, 3)
+    env_fn = lambda: gym.make('PongNoFrameskip-v4')
+    env1 = env_fn()
+    env2 = AtariPreprocessing(env_fn(), screen_size=84, grayscale_obs=True, frame_skip=1, noop_max=0)
+    env3 = AtariPreprocessing(env_fn(), screen_size=84, grayscale_obs=False, frame_skip=1, noop_max=0)
+    env1.reset()
+    # take these steps to imitate actions of FireReset logic
+    env1.step(1)
```

Comment:
per discussion here: https://github.com/openai/baselines/issues/240 it may also make sense to remove FireReset logic altogether